android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -1904,6 +1909,11 @@
 	 */
 	public static final String PROPERTY_LOAD_URL = "loadUrl";
 
+	/**
+	 * @module.api
+	 */
+	public static final String PROPERTY_LOADED = "loaded";
```

Comment:
Titanium's JavaScript events typically do not use past-tence (ex: 'ed) or future-tense (ex: 'ing) names. I know, I prefer to write them your way too, but we should be consistent with the rest of the framework.

Our 'Ti.UI.ImageView' type has a `load` event to indicate that an image has finished loaded asynchronously. We may want to do the same for the toolbar.
http://docs.appcelerator.com/platform/latest/#!/api/Titanium.UI.ImageView-event-load

Reply:
Removing this completely. It is a leftover from older commits.